Sprint-2/3-mandatory-implement/1-bmi.js

```diff
@@ -15,5 +15,10 @@
 // It should return their Body Mass Index to 1 decimal place
 
 function calculateBMI(weight, height) {
-    // return the BMI of someone based off their weight and height
-}
\ No newline at end of file
+  const bmi = weight / (height * height);
+  // return the BMI of someone based off their weight and height
+  return parseFloat(bmi.toFixed(1));
```

Comment:
Well done, you can even enhance this fn by also returning  result about people's bmi . Consider this as a challenge and if you have time go for it. 
![image](https://github.com/user-attachments/assets/31fa8fa3-e448-481d-82f6-a7ca84462815)